packages/remix-dev/devServer_unstable/index.ts

```diff
@@ -267,6 +268,12 @@ export let serve = async (
 
   server.listen(options.port);
 
+  exitHook(() => {
+    state.appServer?.pid && kill(state.appServer.pid);
```

Comment:
probably want to call `state.appServer.kill()` instead to let the app server terminate gracefully